### DISCUSSION TITLE
Module parse failed: Unexpected token on coinbaseWallet.js

### DISCUSSION THREAD

**Discussion by saajand** [wevm/wagmi#1997] [Q&A] [answered]
Hey, so I am trying to work with **Rainbowkit** and **Wagmi** on my react project but seem to get Module parse fail error.
I tried different versions for wagmi but still getting the issue.
Currently running the latest "@rainbow-me/rainbowkit": "^0.12.1" and "wagmi": "^0.12.1".

```
./node_modules/@wagmi/connectors/dist/coinbaseWallet.js 108:81
Module parse failed: Unexpected token (108:81)
File was processed with these loaders:
 * ./node_modules/react-scripts/node_modules/babel-loader/lib/index.js
You may need an additional loader to handle the result of these loaders.
|       class WalletProvider {}
|       class Client {}
>       const walletExtensionChainId = __privateGet(this, _client).walletExtension?.getChainId();
|       const chain = this.chains.find(chain2 => this.options.chainId ? chain2.id === this.options.chainId : chain2.id === walletExtensionChainId) || this.chains[0];
|       const chainId = this.options.chainId || chain?.id;
```

I was able to make it work successfully on another demo project but even after installing the same versions of raibowkit, wagmi and ethers, I am unable to make it work on this project. I tried upgrading react-scripts to "5.0.1" but still got the same error above somehow.

Both projects were created using CRA, demo project running on react-scripts "5.0.1" and the main project running on react-scripts "4.0.3".

Any help is appreciated. Thanks!

**Comment by tmm** [comment]
CRA is not officially supported by wagmi. This looks like a babel issue. You might need to update your package.json#browserlist.

**Comment by saajand** [comment, marked as answer]
Thanks @tmm for the response.
For anyone experiencing the same issue, I fixed this by following these steps:

1. Updated the browserlist to
```
"browserslist": [
  ">0.2%",
  "not dead",
  "not op_mini all"
],
```

2. Cleared npm
`npm cache clean --force`

3. Then reinstalled after removing the node_modules
`rm -rf node_modules`
`npm install`
